### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by MohamedSy1**
I was able to add dark mode for the website

### CODE REVIEW

File: website/src/index.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 :root {
-  --white: hsla(0deg 0% 100% / 100%);
+  --white: #fff;
```
Comment (Idrinth) [inline website/src/index.css:2]: please return this to hsla :)

File: website/src/index.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   --light-gray: hsla(0deg 0% 95% / 100%);
   --dark-gray: hsla(0deg 0% 25% / 100%);
   --dark-green: hsla(132deg 33% 29% / 100%);
   --green: hsla(120deg 77% 36% / 100%);
-  --black: hsla(0deg 0% 0% / 100%);
+  --black: hsla(0deg 0% 0% / 100%);
```
Comment (Idrinth) [inline website/src/index.css:7]: please remove the added whitespace

File: website/src/index.css [hunk 3]
```diff
@@ -224,6 +224,28 @@ footer li:last-child::after {
...
+      --dark-gray: var(#00100d);
+      --dark-green: var(#021f07);
+
+      background-color: var(#090807);
+      color: var(#fff);
+    }
+
+    a {
+      color: var(#008e00);
+      text-decoration: underline;
```
Comment (Idrinth) [inline website/src/index.css:239]: not a colot, please remove

File: website/src/index.css [hunk 4]
```diff
@@ -224,6 +224,28 @@ footer li:last-child::after {
...
+    }
+    
+    nav a {
+      font-size: 130%;
+      color: var(#008e00);
+      text-decoration: none;
```
Comment (Idrinth) [inline website/src/index.css:245]: not a color, please remove

File: website/src/index.css [hunk 5]
```diff
@@ -224,6 +224,28 @@ footer li:last-child::after {
...
   }
 
+  @media (prefers-color-scheme: dark) {
+    :root {
+      --light-gray: var(#4b4b4b);
+      --dark-gray: var(#00100d);
+      --dark-green: var(#021f07);
+
+      background-color: var(#090807);
+      color: var(#fff);
```
Comment (Idrinth) [inline website/src/index.css:234]: don't those belong in the body?

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by MohamedSy1** [comment]
Done

File: website/src/index.css [hunk 6]
```diff
@@ -231,6 +231,27 @@ footer li:last-child::after {
...
+      --light-gray: #4b4b4b;
+      --dark-gray: #001000;
+      --light-green: #026512;
+      
+      background-color: #090807;
+      color: #d8d6d6;
+    }
+    
+    nav a {
+      font-size: 130%;
```
Comment (Idrinth) [inline website/src/index.css:245]: not a color, please remove

File: website/src/index.css [hunk 7]
```diff
@@ -231,6 +231,27 @@ footer li:last-child::after {
     text-align: left;
   }
 
+  @media (prefers-color-scheme: dark) {
+    :root {
+      --light-gray: #4b4b4b;
+      --dark-gray: #001000;
+      --light-green: #026512;
+      
+      background-color: #090807;
```
Comment (Idrinth) [inline website/src/index.css:240]: why are those in root? Shouldn't they be in html or body?

File: website/src/index.css [hunk 8]
```diff
@@ -1,10 +1,10 @@
 :root {
-  --white: hsla(0deg 0% 100% / 100%);
+  --white: hsla(0deg 0% 95% / 100%);
```
Comment (Idrinth) [inline website/src/index.css:2]: why the change by 5%?

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by Idrinth** [comment]
I think we need to go through this together. The merge broke a lot of the css by deleting large parts of it, then there are generated files being added and lastly a new package being added to the website package.json.

Let me know when you're available.

**Review by Idrinth — APPROVED** [review]

**Pull request merged**